Source/DiscoveryV1/Discovery.swift

```diff
@@ -2120,6 +2120,10 @@ public class Discovery {
        **date** or **number** format. When a **date** type field is specified returned results are biased towards field
        values closer to the current date. When a **number** type field is specified, returned results are biased towards
        higher field values. This parameter cannot be used in the same query as the **sort** parameter.
+     - parameter spellingSuggestions: When `true` and the **natural_language_query** parameter is used, the
+       **natural_languge_query** parameter is spell checked. The most likely correction is retunred in the
+       **suggested_query** field of the response (if one exists).
+       **Important:** this parameter is only valid when using the Cloud Pak version of Discovery.
```

Comment:
Why are there changes to Discovery in this PR?  Can you eliminate these so that the PR is only about adding VRv4?